test/e2e/network/netpol/test_helper.go

```diff
@@ -30,7 +30,8 @@ import (
 )
 
 const (
-	waitInterval = 1 * time.Second
+	// increased to 5 seconds, from 1, so that the test waits longer to start in case of congestion
```

Comment:
note ~ empirically, we're finding that these new tests run 30% faster, so this isnt actually
slowing anything down.

Reply:
@rajaskakodkar , can you make the call on wether or not we stil need this?

Reply:
I think we can drop this